patches/minecraft/net/minecraft/entity/EntityCreature.java.patch

```diff
@@ -0,0 +1,28 @@
+--- ../src-base/minecraft/net/minecraft/entity/EntityCreature.java
++++ ../src-work/minecraft/net/minecraft/entity/EntityCreature.java
+@@ -15,6 +15,7 @@
+     private BlockPos field_70775_bC = BlockPos.field_177992_a;
+     private float field_70772_bD = -1.0F;
+     private final float field_184661_bw = PathNodeType.WATER.func_186289_a();
++    protected static float maxLeashDistance = 10.0F;
```

Comment:
no need for static, I could see this being useful for modders who want different values for different creature types

Reply:
Probably add a comment as well

Reply:
@williewillus Yeah I was already sceptical about that but didn't change it. Will do in a minute.
@liach What do you mean? What would be in that comment?

Reply:
Something like `// Forge: added leash distance` so that future field adders may just add next to your patch, reducing total patch size